README.md

```diff
@@ -2,7 +2,135 @@
+- `--min_mapping_percent_cov`: The total percentage of positions within the genome that must have a depth of coverage greater than the minimum depth of coverage specified in order to pass the mapping quality check and be included in the analysis.
+- `--min_mean_mapping_quality`: The minimum mean mapping quality score for all reads in a pileup to be included in the analysis.
+- `--snv_abundance_ratio`: The proportion of reads required to support a variant to be included in the analysis.
+- `--min_repeat_length`: The minimum length when identifying repeats.
+- `--min_repeat_pid`: The minimum percent identity when identifying repeats.
+- `--skip_density_filter`: Whether or not to skip filtering low SNV density regions.
+
+Please refer to the [SNVPhyl documentation](https://snvphyl.readthedocs.io/en/latest/) for more detailed information about pipeline parameters.
+
+# Running
+
+## Test Data
+
+In order to run the pipeline with provided data, please run:
+
+```
+nextflow run main.nf -profile singularity --input https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv --refgenome https://raw.githubusercontent.com/phac-nml/snvphylnfc/main/assets/reference.fasta --outdir results
+```
+
+The pipeline output will be written to a directory named `results`. A JSON file for integrating data with IRIDA Next will be written to `results/iridanext.output.json.gz` (please see the Output section for details).
+
+# Output
+
+## Results
+
+The following output files are generated by the pipeline:
+
+- `make/snvMatrix.tsv`: a pair-wise distance matrix of SNVs that passed all filtering criteria
+- `filter/filterStats.txt`: a summary of the number of SNVs filtered within in the SNV Table
+- `phyml/phylogeneticTree.newick`: the maximum likelihood phylogeny generated from an alignment of SNVs extracted from the whole genomes of each input file
+- `phyml/phylogeneticTreeStats.txt`: statistics for the generated phylogenetic tree
+- `vcf2snv/snvTable.tsv`: a table of all detected variant sites
+- `vcf2snv/vcf2core.tsv`: a table of the evaluated core positions in each reference fasta sequence
+- `vcf2snv/snvAlignment.phy`: an alignment of SNVs used to generate the phylogenetic tree
+- `verifying/mappingQuality.txt`: describes how well the given reads mapped to the reference genome
+
+For more detailed information, please refer to the [SNVPhyl Documentation](https://snvphyl.readthedocs.io/en/latest/user/output/).
+
+## IRIDA Next Integration File
+
+A JSON file for loading the data into IRIDA Next is output by this pipeline. The format of this JSON file is specified in our [Pipeline Standards for the IRIDA Next JSON](https://github.com/phac-nml/pipeline-standards#32-irida-next-json). This JSON file is written directly within the `--outdir` provided to the pipeline with the name `irida.output.json.gz` (ex: `[outdir]/irida.output.json.gz`).
```

Comment:
Name **iridanext.output.json.gz**

Reply:
Updated in d47cead498a1fe3ccccfc06977f4a9a19f97afd2